SharePortfolioManager/Classes/ShareObjects/ShareObjectFinalValue.cs

```diff
@@ -21,7 +21,7 @@
 //SOFTWARE.
 
 // Define for DEBUGGING
-#define DEBUG_FINAL_SHARE_OBJECT
+//#define DEBUG_FINAL_SHARE_OBJECT
```

Comment:
This debugging define must be deactivated before the merge to the master branch.